tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +127,18 @@ describe('run', () => {
     });
   });
 
+  it('passes -jsconsole when --browser-console is specified', () => {
+    const firefox = '/pretend/path/to/Firefox/firefox-bin';
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+
+    return cmd.run({firefox} && {browserConsole: true}).then(() => {
```

Comment:
this should be just `return cmd.run(`{browserConsole: true}`).then(...)` (and `{firefox} && {browserConsole: true}` is actually exactly equal to just `{browserConsole: true}`, because `{firefox}` is considered [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and so the expression will be evaluated as the right side of the `... && ...` expression).